playwright-tests/tests/proposal/proposals.spec.js

```diff
@@ -456,6 +456,60 @@ test.describe("Wallet is connected", () => {
     await pauseIfVideoRecording(page);
   });
 
+  test("should show relevant users in mention autocomplete", async ({
+    page,
+  }) => {
+    await page.goto("/devhub.near/widget/app?page=proposal&id=112");
+
+    await page.waitForSelector(`iframe`, {
+      state: "visible",
+    });
+
+    const comment = page.getByRole("link", { name: "geforcy.near" });
+    await comment.waitFor();
+    await comment.scrollIntoViewIfNeeded();
+
+    const heading = page.getByRole("heading", { name: "Relevant Mentions" });
+    await heading.waitFor();
+    await heading.scrollIntoViewIfNeeded();
+
+    await page.waitForTimeout(5000);
+
+    const delay_milliseconds_between_keypress_when_typing = 0;
+    const commentEditor = page
+      .frameLocator("iframe")
+      .locator(".CodeMirror textarea");
+    await commentEditor.focus();
+    await commentEditor.pressSequentially(
+      `Make sure relevant users show up in a mention. @`,
+      {
+        delay: delay_milliseconds_between_keypress_when_typing,
+      }
+    );
+
+    await pauseIfVideoRecording(page);
+    const iframe = page.frameLocator("iframe");
+    const liFrameLocators = iframe.frameLocator(
+      'ul[id="mentiondropdown"] > li'
+    );
+    const liLocators = await liFrameLocators.owner().all();
+    const expected = [
+      "thomasguntenaar.near", // author,
+      "theori.near", // supervisor,
+      "neardevdao.near", //  requested_sponsor,
+      // "geforcy.near", // comment author,
+    ];
+    let mentionSuggestions = [];
+    for (let i = 0; i < liLocators.length; i++) {
+      const text = await liLocators[i].innerText();
+      mentionSuggestions.push(text);
+    }
+    // TODO should be 0,4 including geforcy.near but somehow mob.near shows up.
+    // When I manually test, it shows the correct 4 users
```

Comment:
so do you want to fix it before merging, so can we merge it?

Reply:
I will change it before we merge. I have an idea on why it happens, we should mock the Social.index call.